app/serializers/serialize_code_tags_sample.rb

```diff
@@ -0,0 +1,28 @@
+class SerializeCodeTagsSample
+  include Mandate
+
+  initialize_with :sample, status: Mandate::NO_DEFAULT
+
+  def call
+    {
+      uuid: sample.uuid,
+      status: sample.status,
+      track: {
+        title: track.title,
+        icon_url: track.icon_url
+      },
+      exercise: {
+        title: exercise.title,
+        icon_url: exercise.icon_url
+      },
+      created_at: sample.created_at,
+      updated_at: sample.updated_at,
```

Comment:
Are these needed? Why do we show `created_at` in the UI?

Reply:
@dem4ron ?